src/usr/share/batwatch/batwatch-daemon.py

```diff
@@ -73,13 +64,15 @@ def get_user_and_group_ids():
     try:
         program_user = pwd.getpwnam(PROCESS_USERNAME)
     except KeyError as key_error:
-        # TODO: When switching to Python 3, convert to chained exception. (gpgmailer issue 15)
+        # TODO: When switching to Python 3, convert to chained exception.
+        #   (gpgmailer issue 15)
         print('User %s does not exist.' % PROCESS_USERNAME)
         raise key_error
     try:
         program_group = grp.getgrnam(PROCESS_GROUP_NAME)
     except KeyError as key_error:
-        # TODO: When switching to Python 3, convert to chained exception. (gpgmailer issue 15)
+        # TODO: When switching to Python 3, convert to chained exception.
+        #   (gpgmailer issue 15)
```

Comment:
I'm confused about why I said this should go on the next line if it doesn't break the 93 character rule. What was I thinking???

Reply:
This would bring the line to 95 characters.

Reply:
Oh! Carry on!